pkg/version/version.go

```diff
@@ -77,3 +79,43 @@ func (v *Build) String() string {
 func (v *Build) ShortVersion() string {
 	return fmt.Sprintf("%s-%s-%s", v.Version, v.BuildNumber, v.GitCommit)
 }
+
+// Equal determines if v is equal to b based on BuildNumber
+func (v *Build) Equal(b *Build) bool {
+	if v.BuildNumber != b.BuildNumber {
+		return false
+	}
+	return true
+}
+
+// IsOlder determines if v is older than b based on BuildNumber
+func (v *Build) IsOlder(b *Build) (bool, error) {
+	if v.Equal(b) {
+		return false, nil
+	}
+
+	if v.BuildNumber == "" || b.BuildNumber == "" {
+		return false, errors.New("invalid BuildNumber")
+	}
+
+	vi, errv := strconv.Atoi(v.BuildNumber)
+	bi, errb := strconv.Atoi(b.BuildNumber)
+	if errv != nil || errb != nil {
+		return false, errors.New("invalid BuildNumber format")
```

Comment:
outputting `errb` and `errv` as part of this message will likely help with debugging if this ever occurs. Likewise it might be good to include information about which was empty in the one on line 98